src/main/java/net/openhft/chronicle/queue/impl/single/QueueLock.java

```diff
@@ -44,4 +45,8 @@ public interface QueueLock extends Closeable {
      * @return true if the lock is locked
      */
     boolean isLocked();
+
+    default boolean forceUnlockIfProcessIsDead() {
```

Comment:
Adding a method to a deprecated interface seems odd, but this is to remove casts elsewhere